docs/ui/components/home/GridCell.tsx

```diff
@@ -0,0 +1,35 @@
+import { css, SerializedStyles } from '@emotion/react';
+import { borderRadius, theme } from '@expo/styleguide';
+import React from 'react';
+import { Col } from 'react-grid-system';
+
+type GridCellProps = {
+  children: any;
+  style: SerializedStyles;
+  [props: string]: any;
+};
+
+export const GridCell = ({ children, style, md }: GridCellProps) => (
+  <Col css={gridCellWrapperStyle} md={md}>
+    <div css={[gridCellStyle, style, !style && gridCellDefaultStyle]}>{children}</div>
```

Comment:
If I read this right, the `gridCellDefaultStyle` should be applied when there is no style provided, right? But the style type isn't optional I think. maybe adding `children, md, style = gridCellDefaultStyle`, and make the prop optional might be a good solution?